vscode/src/workspace.ts

```diff
@@ -189,7 +189,7 @@ export class Workspace implements WorkspaceInterface {
   }
 
   async stop() {
```

Comment:
Should this be called `dispose` instead 🤔

Reply:
We also have a dispose, but they're not quite the same thing. When we stop a workspace, it means stopping the LSP client (which we do dispose), but we don't dispose of the workspace itself. That remains active until the extension is deactivated.